src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CalibrationWindow.cs

```diff
@@ -0,0 +1,185 @@
+
+            if (intrinsics.IntrinsicsFileName != string.Empty &&
+                intrinsics.Intrinsics != null)
+            {
+                RenderTitle(intrinsics.IntrinsicsFileName, Color.green);
+                GUILayout.Label($"Reprojection Error: {intrinsics.Intrinsics.ReprojectionError.ToString("G4")}");
+                GUILayout.Label($"Focal Length: {intrinsics.Intrinsics.FocalLength.ToString("G4")}, Principal Point: {intrinsics.Intrinsics.PrincipalPoint.ToString("G4")}");
+                GUILayout.Label($"Radial Distortion: {intrinsics.Intrinsics.RadialDistortion.ToString("G4")}, Tangential Distortion: {intrinsics.Intrinsics.TangentialDistortion.ToString("G4")}");
+            }
+        }
+
+        private void UpdateExtrinsicsUI(EditorExtrinsicsCalibration extrinsics)
+        {
+            RenderTitle("Camera Extrinsics", Color.green);
+            GUILayout.Label($"Usable marker datasets: {extrinsics.ProcessedDatasets}");
+            GUILayout.Label($"Number of detected markers in last dataset: {extrinsics.LastCountDetectedMarkers}, minimum required: {extrinsics.MinimumNumberOfDetectedMarkers}");
+            EditorGUILayout.Space();
+
+            GUIStyle boldLabelStyle = new GUIStyle(GUI.skin.label);
+            boldLabelStyle.fontStyle = FontStyle.Bold;
+
+            RenderTitle(HolographicCameraDeviceTypeLabel, Color.green);
+            var cameraDevice = GetHolographicCameraDevice();
+            if (cameraDevice != null && cameraDevice.NetworkManager != null && cameraDevice.NetworkManager.IsConnected)
+            {
+                GUILayout.BeginHorizontal();
+                {
+                    GUILayout.Label("Connection status", boldLabelStyle);
+
+                    if (GUILayout.Button(new GUIContent("Disconnect", "Disconnects the network connection to the holographic camera."), GUILayout.Width(connectAndDisconnectButtonWidth)))
+                    {
+                        cameraDevice.NetworkManager.Disconnect();
+                    }
+                }
+                GUILayout.EndHorizontal();
+
+                if (cameraDevice.NetworkManager.ConnectedIPAddress == cameraDevice.DeviceIPAddress)
+                {
+                    GUILayout.Label($"Connected to {cameraDevice.DeviceName} ({cameraDevice.DeviceIPAddress})");
+                }
+                else
+                {
+                    GUILayout.Label($"Connected to {cameraDevice.DeviceName} ({cameraDevice.NetworkManager.ConnectedIPAddress} -> {cameraDevice.DeviceIPAddress})");
+                }
+            }
+            else
+            {
+                GUILayout.BeginHorizontal();
+                {
+                    holographicCameraIPAddress = EditorGUILayout.TextField(holographicCameraIPAddress);
+                    ConnectButtonGUI(holographicCameraIPAddress, cameraDevice);
+                }
+                GUILayout.EndHorizontal();
+
+                GUILayout.Label(notConnectedMessage);
+            }

```

Comment:
I think this is almost entirely copy-pasted from the base class, can we extract this into its own method in the base class and then you could just call that method from here? I'm thinking it could be called something like "ConnectionStatusGUI"